<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>162</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RC</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="refreshButton">
     <property name="text">
      <string>Обновить</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="refreshLabel">
     <property name="text">
      <string>Обновление...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Список серверов:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="serverList"/>
   </item>
   <item>
    <widget class="QPushButton" name="connectButton">
     <property name="text">
      <string>Присоедениться</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
